tox.ini

```diff
@@ -46,7 +46,7 @@ deps =
     -r{toxinidir}/requirements.txt
     mock
     pytest-cov
-    pytest==3.5.1
```

Comment:
Maybe use >= here to make it clear which versions are required.

Reply:
Yeah. That's better, I will use it.

Reply:
@EdDev wants to hardcode the versions